packages/component/src/loadable.test.js

```diff
@@ -242,6 +279,21 @@ describe('#lazy.lib', () => {
     const library = { it: 'is', a: 'lib' }
     load.resolve(library)
     await wait(() => expect(container).toHaveTextContent('loaded'))
-    expect(container).toHaveTextContent('loaded')
+  })
+
+  it('supports Error Boundary', async () => {
+    const load = createLoadFunction()
+    const Lib = lazy.lib(load)
+    const renderFn = jest.fn(() => 'loaded')
+    const { container } = render(
+      <ErrorBoundary fallback="error">
+        <React.Suspense fallback="progress">
+          <Lib>{renderFn}</Lib>
+        </React.Suspense>
+      </ErrorBoundary>,
+    )
+    expect(container).toHaveTextContent('progress')
+    load.reject(new Error('Error Boundary'))
+    await wait(() => expect(container).toHaveTextContent('error'))
```

Comment:
Please add one more test - `supports retry from the Error Boundary`. It shall not be a final, unrecoverable, state.